v19.1/bytes.md

```diff
@@ -66,6 +66,41 @@ valid UTF-8 byte sequences; an error is reported otherwise.
 `STRING` values can be cast explicitly to `BYTES`. This conversion
 always succeeds.
 
+### `STRING` vs. `BYTES`
+
+While both `STRING` and `BYTES` can appear to have similar behavior in many situations, one should understand their nuance before casting one into the other.
+
+`STRING` treats all of its data as characters, or more specificially, unicode code points. `BYTES` treats all of its data as a byte string. This difference in implementation can lead to dramatically different behavior. For example, let's take a complex ASCII character such as ☃ ([the snowman emoji](https://emojipedia.org/snowman/)):
+
+{% include copy-clipboard.html %}
+~~~ sql
+SELECT length('☃'::string);
+~~~
+
+~~~
+  length
++--------+
+       1
+~~~
+
+~~~ sql
+SELECT length('☃'::bytes);
+~~~
+~~~
+  length
++--------+
+       3
+~~~
+
+In this case, `LENGTH(string)` measures the number of unicode code points present in the string, where as `LENGTH(bytes)` measures the number of bytes required to store that value. Each character (or unicode code point) can be encoded using multiple bytes, hence the difference in output between the two.
+
+#### Translating literals to `STRING` vs. `BYTES`
+
+A literals entered through a SQL client will be translated into a different value based on the type:
+
++ `BYTES` give a special meaning to the pair `\x` at the beginning, and translate the rest by substituting pairs of hexadecimal digits to a single byte. For example, `\xff` is equivalent to a single byte with the value of 255.
```

Comment:
I think 'translate' should be plural here, i.e., "`BYTES` ... translates the rest ..."